gnomad/resources/grch38/gnomad.py

```diff
@@ -513,9 +515,12 @@ def gnomad_gks_batch(
             " please also specify 'by_ancestry_group' to stratify by."
         )
 
-    # Call and return add_gks*() for chosen arguments.
+    # Call and return add_gks_vrs and add_gks_va for chosen arguments.
 
     # Filter to interval before adding annotations
+    ht = ht.select(ht.freq, ht.info.vrs, ht.popmax)
+    if checkpoint_path:
+        ht = ht.checkpoint(checkpoint_path, overwrite=overwrite)
```

Comment:
rather than adding in two new params, you can change this to just use new_temp_file